pkg/certs/cert-inspection/certgraphanalysis/deduplication.go

```diff
@@ -56,28 +81,44 @@ func combineCertOnDiskLocations(in *certgraphapi.CertKeyPair, rhs []certgraphapi
 			}
 		}
 		if !found {
+			// Store key to be merged into Cert
+			if len(curr.Cert.Path) == 0 {
```

Comment:
I still don't understand why we want to skip this.  if it's not equal, just add it